Resources/Prototypes/BLUESPACE/RHBZZ.yml

```
@@ -0,0 +1,249 @@
+  name: ключ шифрования РХБЗЗ
+  description: Ключ шифрования для связи РХБЗЗ.
+  components:
+  - type: EncryptionKey
+    channels:
+    - RHBZZCommunication
+    defaultChannel: RHBZZCommunication
+  - type: Sprite
+    layers:
+    - state: crypt_blue
+    - state: nano_label
+
+- type: entity
+  parent: ClothingHeadset
+  id: ClothingHeadsetRHBZZ
+  name: гарнитура РХБЗЗ
+  description: Гарнитура для элитного подразделения РХБЗЗ.
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
+      - EncryptionKeyRHBZZ
+  - type: Sprite
+    sprite: Clothing/Ears/Headsets/RHBZZ.rsi
+  - type: Clothing
+    sprite: Clothing/Ears/Headsets/RHBZZ.rsi
+
+
+
+#Объекты:
+
+
+#обувь
+
+- type: entity
+  parent: ClothingShoesBaseButcherable
+  id: ClothingShoesRHBZZ
+  name: сапоги РХБЗЗ
+  description: Защищает ноги от биологической угрозы.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/RHBZZ.rsi
+    state: icon
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/RHBZZ.rsi
+  - type: Magboots
+    toggleAction: ActionToggleMagbootsAdvanced
+  - type: ClothingSpeedModifier
+    walkModifier: 1.1
+    sprintModifier: 1.3
+    enabled: false
+  - type: NoSlip
+  - type: FootstepModifier
+    footstepSoundCollection:
+      path: /Audio/Effects/Footsteps/RHBZZ.ogg
+      params:
+        volume: 3
+  - type: Tag
+    tags:
+    - FootstepSound
+
+
+#очки
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesGlassesRHBZZ
+  name: очки РХБЗЗ
+  description: Профессиональные очки ДСО, предназначенный для лучшего обнаружения гуманоидов.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Hud/RHBZZ.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Hud/RHBZZ.rsi
+  - type: ShowSecurityIcons
+  - type: FlashImmunity
+  - type: EyeProtection
+
+
+#перчатки
+
+- type: entity
+  parent: ClothingHandsBase
+  id: ClothingHandsGlovesRHBZZ
+  name: резиновые перчатки РХБЗЗ
+  description: Высококачественные резиновые перчатки, предназначены для борьбы с биологическими угрозами любого уровня!
+  components:
+  - type: Sprite
+    sprite: Clothing/Hands/Gloves/RHBZZ.rsi
+  - type: Clothing
+    sprite: Clothing/Hands/Gloves/RHBZZ.rsi
+  - type: Insulated
+  - type: Fiber
+    fiberMaterial: fibers-rubber
+    fiberColor: fibers-yellow
+  - type: FingerprintMask
+
+
+#униформа
+
+#униформа специалиста РХБЗЗ
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitRHBZZSpecialist
+  name: униформа специалиста РХБЗЗ
+  description: Он изготовлен из специального волокна, обеспечивающего особую защиту от биологических опасностей. На нем есть нашивка специалиста РХБЗЗ.
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpsuit/RHBZZ-jumpsuit.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpsuit/RHBZZ-jumpsuit.rsi
+
+#униформа коммандера РХБЗЗ
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitRHBZZCommander
+  name: униформа коммандера РХБЗЗ

```

Лучше всё-таки командира

```suggestion
  name: униформа командира РХБЗЗ
```